src/atoms/api.ts

```diff
@@ -1,5 +1,7 @@
 import { atom } from "jotai";
 import { getApiClient } from "../api/api";
 import { authAtom } from "./auth";
+import { getLambdasClient } from "../api/home-lambdas-api";
```

Comment:
After making changes in the ./auth as described we should be able to import the getLambdasClient from there rather than from here.